frontend/src/components/RequestDialog.vue

```diff
@@ -122,26 +116,69 @@ const submitRequestTransaction = async (formResult: RequestFormResult) => {
     throw new Error('No signer available!');
   }
 
-  await runTransfer(
-    formResult,
-    provider.value,
-    signer.value,
-    requestManagerAddress.value,
-    requestFeeAmount.value,
-    configuration.chains,
+  const sourceChainConfiguration = configuration.chains[formResult.sourceChainId.value];
+  const sourceChain = {
+    identifier: sourceChainConfiguration.identifier,
+    name: sourceChainConfiguration.name,
+    rpcUrl: sourceChainConfiguration.rpcUrl,
+    requestManagerAddress: sourceChainConfiguration.requestManagerAddress,
+    fillManagerAddress: sourceChainConfiguration.fillManagerAddress,
+    explorerTransactionUrl: sourceChainConfiguration.explorerTransactionUrl,
+  };
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const sourceToken = sourceChainConfiguration.tokens.find(
+    (token) => token.symbol === formResult.tokenAddress.label,
+  )!;
+
+  const targetChainConfiguration = configuration.chains[formResult.targetChainId.value];
+  const targetChain = {
```

Comment:
Same as the `sourceChain`

Reply:
I'll close this conversation in favor of [this one](https://github.com/beamer-bridge/beamer/pull/651#discussion_r874877277).